src/ports/postgres/modules/dbscan/test/perf/results.txt

```diff
@@ -0,0 +1,14 @@
+2D 3C 1000 points:
```

Comment:
Do we need this file? seems like these files that were not intended got checked in.